src/legacy/ui/public/field_editor/components/scripting_help/__snapshots__/help_flyout.test.tsx.snap

```diff
@@ -0,0 +1,139 @@
+                  "painless": "https://www.elastic.co/guide/en/elasticsearch/reference/mocked-test-branch/modules-scripting-painless.html",
+                  "painlessApi": "https://www.elastic.co/guide/en/elasticsearch/painless/mocked-test-branch/painless-api-reference.html",
+                  "painlessSyntax": "https://www.elastic.co/guide/en/elasticsearch/reference/mocked-test-branch/modules-scripting-painless-syntax.html",
+                  "scriptAggs": "https://www.elastic.co/guide/en/elasticsearch/reference/mocked-test-branch/search-aggregations.html#_values_source",
+                  "scriptFields": "https://www.elastic.co/guide/en/elasticsearch/reference/mocked-test-branch/search-request-script-fields.html",
+                }
+              }
+            />,
+            "data-test-subj": "syntaxTab",
+            "id": "syntax",
+            "name": "Syntax",
+          },
+          Object {
+            "content": <TestScript
+              executeScript={[Function]}
+              getHttpStart={[Function]}
+              indexPattern={Object {}}
+              lang="painless"
+            />,
+            "data-test-subj": "testTab",
+            "id": "test",
+            "name": "Preview results",
+          },
+        ]
+      }
+    />
+  </EuiFlyoutBody>
+</EuiFlyout>
+`;
+
+exports[`ScriptingHelpFlyout should render nothing if not visible 1`] = `
+<EuiFlyout
+  closeButtonAriaLabel="Closes this dialog"
+  data-test-subj="scriptedFieldsHelpFlyout"
+  hideCloseButton={false}
+  maxWidth={false}
+  onClose={[Function]}
+  ownFocus={false}
+  size="m"
+>
+  <EuiFlyoutBody>
+    <EuiTabbedContent
+      autoFocus="initial"
+      initialSelectedTab={
+        Object {
+          "content": <ScriptingSyntax
+            docLinksScriptedFields={
+              Object {
+                "luceneExpressions": "https://www.elastic.co/guide/en/elasticsearch/reference/mocked-test-branch/modules-scripting-expression.html",
+                "painless": "https://www.elastic.co/guide/en/elasticsearch/reference/mocked-test-branch/modules-scripting-painless.html",
+                "painlessApi": "https://www.elastic.co/guide/en/elasticsearch/painless/mocked-test-branch/painless-api-reference.html",
+                "painlessSyntax": "https://www.elastic.co/guide/en/elasticsearch/reference/mocked-test-branch/modules-scripting-painless-syntax.html",
+                "scriptAggs": "https://www.elastic.co/guide/en/elasticsearch/reference/mocked-test-branch/search-aggregations.html#_values_source",
+                "scriptFields": "https://www.elastic.co/guide/en/elasticsearch/reference/mocked-test-branch/search-request-script-fields.html",
+              }
+            }
+          />,
+          "data-test-subj": "syntaxTab",
+          "id": "syntax",
+          "name": "Syntax",
+        }
+      }
+      tabs={
+        Array [
+          Object {
+            "content": <ScriptingSyntax
+              docLinksScriptedFields={
```

Comment:
Maybe in this file, too, we can stub `docLinksScriptedFields` as empty object `{}`.